src/etl/LoadBalancer.cpp

```diff
@@ -379,11 +406,15 @@ LoadBalancer::forwardToRippledImpl(
     std::optional<boost::json::object> response;
     rpc::ClioError error = rpc::ClioError::EtlConnectionError;
     while (numAttempts < sources_.size()) {
-        auto res = sources_[sourceIdx]->forwardToRippled(request, clientIp, xUserValue, yield);
+        auto [res, duration] =
+            util::timed([&]() { return sources_[sourceIdx]->forwardToRippled(request, clientIp, xUserValue, yield); });
+        forwardedDurationHistogram_.get().observe(duration);
```

Comment:
Maybe we should separate successful and failed requests here.

Reply:
Waiting for the response above first